src/tests/test_magic.py

```diff
@@ -2597,3 +2597,19 @@ def test_table_does_not_exist_with_snippet_error(
     # Test error and message
     assert error_type == excinfo.value.error_type
     assert error_message in str(excinfo.value)
+
+
+@pytest.mark.parametrize(
+    "query",
+    [
+        "%sql select 5 * -2",
+        "%sql select 5 * - 2",
+        "%sql select 5 * -2;",
```

Comment:
you can also add some test case like numeric column of a test table * (some negative constant).  You can use the test `number_table`

Reply:
[Added these test cases](https://github.com/ploomber/jupysql/pull/964/files#diff-d327143929530bcece2cae65d59465708cdf87f499fd83b4bd939c332150f713R2611-R2638)